clang-tools-extra/unittests/clang-doc/HTMLMustacheGeneratorTest.cpp

```diff
@@ -86,8 +87,16 @@ TEST(HTMLMustacheGeneratorTest, generateDocs) {
   assert(G && "Could not find HTMLMustacheGenerator");
   ClangDocContext CDCtx = getClangDocContext();
 
-  StringRef RootDir = "";
-  EXPECT_THAT_ERROR(G->generateDocs(RootDir, {}, CDCtx), Succeeded())
+  unittest::TempDir RootTestDirectory("generateDocsTest", /*Unique=*/true);
+  CDCtx.OutDirectory = RootTestDirectory.path();
+
+  // This seems wrong, but its unclear how else we would test this...
+  SmallString<128> AssetPath("../../../../../share/clang-doc");
+  llvm::sys::path::native(AssetPath);
+  getMustacheHtmlFiles(AssetPath, CDCtx);
```

Comment:
This whole thing is such a big hack. I'm like 90% sure it will break outside of a Ninja build, and already seems to be tripping up Windows bots. I need to figure out some other way to get the template files loaded.

Reply:
Could you get the path from CMake? What I have in mind is generate a `config.h` file akin to what we do in [Clang](https://github.com/llvm/llvm-project/blob/52f568dbbb61ffe26b7973b482e0e504b405a0ab/clang/CMakeLists.txt#L886-L888) and other LLVM projects and put the relevant paths there.

Reply:
I think I've done this correctly. It's at least working locally. Happy to amend if there's something a bit off.